text/text_test.go

```diff
@@ -0,0 +1,199 @@
+	Convey("Given a metadata struct", t, func() {
+		m := &dataset.Metadata{
+			Version: dataset.Version{
+				ReleaseDate: "01-02-2006 15:04:05",
+				Dimensions: []dataset.VersionDimension{
+					{
+						ID:                   "id",
+						Label:                "Label 1",
+						Description:          "description",
+						NumberOfOptions:      1,
+						QualityStatementText: "quality statement",
+					},
+					{
+						ID:                   "id",
+						Label:                "Label 2",
+						Description:          "description",
+						NumberOfOptions:      1,
+						QualityStatementText: "quality statement",
```

Comment:
Similarly, can we make sure we have different values so that we know we are reading them from the right place?